server/src/main/java/io/unitycatalog/server/service/AuthService.java

```diff
@@ -146,6 +150,29 @@ public HttpResponse grantToken(OAuthTokenExchangeRequest request) {
     return HttpResponse.ofJson(response);
   }
 
+  private static void verifyPrincipal(DecodedJWT decodedJWT) {
+    String subject =
+        decodedJWT.getClaim("email").isMissing()
+            ? decodedJWT.getClaim("sub").asString()
+            : decodedJWT.getClaim("email").asString();
+
+    if (subject.equals("admin")) {
```

Comment:
This is temporary... when access control is done, `admin` will be a real user in. the db and i'll remove this.